project/project.go

```diff
@@ -337,6 +383,16 @@ func (p *Project) Down(ctx context.Context, opts options.Down, services ...strin
 		return err
 	}
 
+	if opts.RemoveVolume {
```

Comment:
Ah, it's already handled for `rm` (but I'll have to check on `docker-compose` to see if it's removing the named volume too)